store/store_test.go

```diff
@@ -6585,6 +6597,7 @@ func (s *storeTestSuite) TestSnapActionRefreshStableInstanceKey(c *C) {
 			TrackingChannel: "stable",
 			Revision:        snap.R(26),
 			RefreshedDate:   helloRefreshedDate,
+			CohortKey:       "what",
```

Comment:
this is a bit weird,  having both a Revision and CohortKey set.  How far are we from realism in the name of test reuse, I know these tests have grown quite a bit since they were created? should we use table tests instead?

Reply:
it's not weird at all: this is context, not action, so if a snap is installed from the store using a cohort key, it will have both set.